hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterDataSerializerHook.java

```diff
@@ -102,8 +103,9 @@
     public static final int TRIGGER_EXPLICIT_SUSPICION = 39;
     public static final int MEMBERS_VIEW_METADATA = 40;
     public static final int HEARTBEAT_COMPLAINT = 41;
+    public static final int PROMOTE_LITE_MEMBER = 42;
 
-    public static final int LEN = HEARTBEAT_COMPLAINT + 1;
```

Comment:
Will this cause any minor version incompatibilities or is the system smart enough not to upgrade an 'old' lite member?